flang/test/Integration/volatile.f90

```diff
@@ -0,0 +1,11 @@
+! RUN: bbc %s -o - | FileCheck %s
```

Comment:
Nit: The integration directory is for testing things that span flang, mlir, llvm. For eg. testing the LLVM IR that is generated. And it should be used sparingly. For this one, the best place is the flang/test/Lower directory.